apps/gateway/lib/gateway/controllers/curse_of_mirra/user_controller.ex

```diff
@@ -38,7 +38,7 @@ defmodule Gateway.Controllers.CurseOfMirra.UserController do
   end
 
   def create_guest_user(conn, %{"client_id" => client_id}) do
-    with {:ok, user} <- Users.create_guest_user() do
+    with {:ok, %{user: user}} <- Users.insert_user_and_insert_daily_quests() do
```

Comment:
Same here. Can't we get the user then insert the daily quests?

Reply:
In this case i don't think that a good idea, you want to either generate a user with everything that the game needs to work or to not generate anything. Since we'll be creating buggy users if we split the user creation from everything else